content/reference/endpoints/rentals_contents_overrides.md

```diff
@@ -1,4 +1,6 @@
 # Rentals Contents Overrides
+Rental content overrides endpoint was made so that different channels could have different length validations for our fields like `headline, summary, description`.
+By using those objects clients applications can use full available length at each channel.
 
```

Comment:
While we are still not sure where to place this I would correct the first part.

Rental content overrides were introduced to address different channels length validations for our fields like `headline, summary, description`.